spec/models/emis_redis/military_information_spec.rb

```diff
@@ -15,6 +27,14 @@
     end
   end
 
+  describe '#service_name_used_in_diability' do
+    it 'something' do
```

Comment:
Could we get a more descriptive test name here?

Reply:
🤦‍♀️absolutely!